test-apps/ui-test-app/src/frontend/appui/statusfields/DisplayStyleField.tsx

```diff
@@ -86,12 +86,16 @@ export class DisplayStyleField extends React.Component<StatusFieldProps, Display
     FrontstageManager.onContentControlActivatedEvent.removeListener(this._handleContentControlActivatedEvent);
   }
 
-  private _handleDisplayStyleSelected = (event: React.ChangeEvent<HTMLSelectElement>): void => {
+  private _handleDisplayStyleSelected = async (event: React.ChangeEvent<HTMLSelectElement>) => {
     if (!this.state.viewport)
       return;
 
     const viewport = this.state.viewport;
-    viewport.displayStyle = this.state.displayStyles.get(event.target.value)!.clone();
+    const style = this.state.displayStyles.get(event.target.value)!.clone();
+    if (style)
```

Comment:
We should probably do the processing in an if style { } block.

Reply:
Done.